packages/react-ui/src/themes/base.js

```diff
@@ -271,7 +271,8 @@ const components = {
   },
   FormLabel: {
     fontSize: 2,
-    marginBottom: 1
+    marginBottom: 1,
+    color: 'text.body'
```

Comment:
No clue why this is here, but I trust you needed it

Reply:
To make the example in constraint page work. The text color was still black even under black theme coming from the body. I thought this was a good default to assume.